cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -27,10 +27,7 @@ metadata:
   labels:
     k8s-app: kube-dns
 spec:
-  # replicas: not specified here:
-  # 1. In order to make Addon Manager do not reconcile this replicas parameter.
-  # 2. Default is 1.
-  # 3. Will be tuned in real time if DNS horizontal auto-scaling is turned on.
+  replicas: {{ .Replicas }}
```

Comment:
In case of `kubeadm init`, the number of replicas for kube-dns isn't specified. Are we changing that here?

Reply:
@rajansandeep will this be solved by 

https://github.com/kubernetes/kubernetes/pull/85837/files#diff-46182be33a6e1280b49fb67c2d22cc58R111